src/Network/Transport/TCP.hs

```diff
@@ -50,6 +50,7 @@ import Network.Transport
 import Network.Transport.TCP.Internal
   ( forkServer
   , recvWithLength
+  , recvWithLengthLimited
```

Comment:
Correct. I'll remove the import, but leave the definition and export since `TestTCP.hs` uses it.